src/apps/HeyGPT/HeyGPT/Activation/ActivationHandlerKeys.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace HeyGPT.App.Activation;
+
+internal class ActivationHandlerKeys
+{
+    public const string DefaultServiceKey = "default";
```

Comment:
When the application is activated the ActivationService will execute multiple handlers.   We have a default handler that handles the UX navigation and the such.   Then there are other handlers for the protocol and file activations.  We want to be able to register these in the DI so that the Default comes separate from the list of other handlers.   We use keyed service registration for that.   These are the key values.